ergo/scale.py

```diff
@@ -71,16 +83,37 @@ class LogScale(Scale):
     log_base: float
 
     def __post_init__(self):
-        # if self.log_base < 1:
-        #     raise ValueError(f"log_Base must be > 1, was {self.log_base}")
         self.width = self.high - self.low
 
     def __hash__(self):
         return super().__hash__()
 
+    def density_denorm(self, true_x):
```

Comment:
Describe what this and `density_norm` are and consider a more descriptive name?

Reply:
I didn't have any more inspired ideas for a name, but I've added descriptions which should help. Take a look and see if, with the description, you think the name is okay.